src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
+	 *
+	 * @param Pages $pages
+	 *
+	 * @return bool
+	 *
+	 */
+	public function purgePages( Pages $pages ) {
+		$pagesArray = $pages->toArray();
+
+		foreach ( $pagesArray as $page ) {
+			$this->purge( $page );
+		}
+
+		return true;
```

Comment:
I think I would be against passing the raw result out of the method. But I don't feel too strongly really.
I think I prefer the true / false based on request success, although I guess to fit with everything else a request that fails would just throw an exception.
maybe void is the way to go?